sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -874,6 +874,37 @@ public async Task GetBatchSettingWithAllFields()
             }
         }
 
+        [TestCase(true)]
+        [TestCase(false)]
```

Comment:
The convention for our tests is to use the [Test] attribute on test methods.  Is there a reason to do it differently here?